tests/phpunit/includes/TestCase.php

```diff
@@ -143,6 +146,38 @@ function () {
 		);
 	}
 
+	protected function fake_site_connection() {
+		add_filter(
+			'googlesitekit_oauth_secret',
+			function () {
+				return json_encode(
+					array(
+						'web' => array(
+							'client_id'     => self::CLIENT_ID,
+							'client_secret' => 'test-client-secret',
+						),
+					)
+				);
+			}
+		);
+	}
+
+	protected function fake_proxy_site_connection() {
```

Comment:
How about we cherry pick the added `Fake_Authentication_Trait` instead?
https://github.com/google/site-kit-wp/blob/71b8bb2fb9fb60bc5ee8cb8166a670c53cc1bc9a/tests/phpunit/includes/Fake_Authentication_Trait.php

Reply:
Good point, will do that.